### ISSUE TITLE
MAINT: Use scikeras.KerasRegressor.r_squared as loss function or catch/assert warning in all tests calling score

### ISSUE THREAD

**Issue by adriangb** [adriangb/scikeras#91]

**Comment by adriangb** [comment]
This warning has been removed. Closing the issue.
